hail/src/main/scala/is/hail/io/plink/LoadPlink.scala

```diff
@@ -45,14 +45,14 @@ object LoadPlink {
     """^-?(?:\d+|\d*\.\d+)(?:[eE]-?\d+)?$""".r
 
   def parseFam(filename: String, ffConfig: FamFileConfig,
-    fs: FS): (IndexedSeq[Row], PStruct) = {
+    fs: FS): (IndexedSeq[Row], PCanonicalStruct) = {
```

Comment:
changed because this allows us to use signature.copy (alternatively the PCanonicalConstructor, which .copy is referencing)

Reply:
That .copy is performed here: https://github.com/hail-is/hail/blob/f098a7f58b9f7dd121cd114ea38ccb52a677dc08/hail/src/main/scala/is/hail/io/plink/LoadPlink.scala#L145

Reply:
yep sounds good